dev/generate

```diff
@@ -2,9 +2,7 @@
 
 set -euo pipefail
 
-rm -rf pkg/proto/**/*.pb.go pkg/proto/**/*.pb.gw.go pkg/proto/**/*.swagger.json
-if ! buf generate https://github.com/xmtp/proto.git#subdir=proto,branch=main; then
-    echo "Failed to generate protobuf definitions"
+if ! ./dev/gen_protos; then
```

Comment:
should `set -e` not handle the fast-fail?

Reply:
You're right, will update